Description: Remove external images from README.md
 They require internet access to be fetched, are not really part
 of the documentation and trigger a privacy-related Lintian warning
Author: Andrea Pappacoda <andrea@pappacoda.it>
Forwarded: not-needed
Last-Update: 2023-02-04

--- tomlplusplus-3.3.0+ds.orig/README.md
+++ tomlplusplus-3.3.0+ds/README.md
@@ -1,9 +1,7 @@
 [![banner](docs/images/banner.png)][homepage]
-[![Releases](https://img.shields.io/github/v/release/marzer/tomlplusplus?style=flat-square)](https://github.com/marzer/tomlplusplus/releases)
 [![C++17](docs/images/badge-C++17.svg)][cpp_compilers]
 [![TOML](docs/images/badge-TOML.svg)][v1.0.0]
 [![MIT license](docs/images/badge-license-MIT.svg)](./LICENSE)
-[![ci](https://github.com/marzer/tomlplusplus/actions/workflows/ci.yaml/badge.svg?branch=master)](https://github.com/marzer/tomlplusplus/actions/workflows/ci.yaml)
 [![Mentioned in Awesome C++](docs/images/badge-awesome.svg)](https://github.com/fffaraz/awesome-cpp)
 [![Sponsor](docs/images/badge-sponsor.svg)](https://github.com/sponsors/marzer)
 [![Gitter](docs/images/badge-gitter.svg)](https://gitter.im/marzer/tomlplusplus)
